tket2/src/rewrite/strategy.rs

```diff
@@ -528,11 +514,11 @@ mod tests {
         let mut circ = n_cx(10);
         let cx_gates = circ.commands().map(|cmd| cmd.node()).collect_vec();
 
-        assert_eq!(circ.rewrite_trace(), None);
```

Comment:
These are just changes to the test cases that expected a `vec`, but now get an `impl Iterator`.